srv/modules/runners/sharedsecret.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+import os
+
+
+def show():
+    filename = '/etc/salt/master.d/sharedsecret.conf'
```

Comment:
I would like to suggest to name the file salt_api_secret.conf to make it clear for what this file is used.